src/ray/protobuf/runtime_env_common.proto

```diff
@@ -66,16 +66,61 @@ message PluginRuntimeEnv {
   repeated Plugin plugins = 1;
 }
 
-/// The jars type runtime env.
-message JarsRuntimeEnv {
-  /// TODO(SongGuyang)
+message PythonRuntimeEnv {
+  /// List of URIs (either in the GCS or external
+  /// storage), each of which is a zip file that will be unpacked and
+  /// inserted into the PYTHONPATH of the workers.
```

Comment:
This will support local files right? We should refine the documentation.
```suggestion
  /// List of URIs that contain the dependent python modules. 
  /// Valid URIs include:
  /// - A local directory: it will be added to to the PYTHONPATH;
  ///   - Example: `file:///path/to/my_module`.
  /// - A zip file in an external storage: it will be unpacked and added to the PYTHONPATH.
  ///   - Example: `http://example.com/my_module.zip`. 
```